api/service/review.service.go

```diff
@@ -0,0 +1,63 @@
+package service
+
+import (
+	"context"
+
+	"shanepee.com/api/domain"
+)
+
+var (
+	ErrReviewNotFound error = domain.ErrReviewNotFound
+)
+
+type ReviewService interface {
+	CreateReview(ctx context.Context, rating int, comment string, artToyID int64) (*domain.Review, error)
+	GetReview(ctx context.Context, artToyID int64) (*domain.Review, error)
+	UpdateReview(ctx context.Context, artToyID int64, updateBody map[string]any) (*domain.Review, error)
+	DeleteReview(ctx context.Context, artToyID int64) error
+}
+
+type reviewServiceImpl struct {
+	reviewRepo domain.ReviewRepository
+}
+
+func NewReviewService(reviewRepo domain.ReviewRepository) ReviewService {
+	return &reviewServiceImpl{
+		reviewRepo,
+	}
+}
+
+var _ ReviewService = &reviewServiceImpl{}
+
+func (s *reviewServiceImpl) CreateReview(ctx context.Context, rating int, comment string, artToyID int64) (*domain.Review, error) {
+	review := domain.NewReview(rating, comment, artToyID)
+	err := s.reviewRepo.CreateReview(ctx, review)
```

Comment:
```suggestion
	err := s.reviewRepo.CreateReview(ctx, review)
```

validate that art toy is purchased and purchased by the use before creating